x-pack/plugins/apm/public/components/app/settings/agent_configurations/agent_configuration_create_edit/service_page/form_row_suggestions_select.tsx

```diff
@@ -23,6 +23,8 @@ interface Props {
   allowAll?: boolean;
   onChange: (value?: string) => void;
   dataTestSubj?: string;
+  isInvalid?: boolean;
+  error?: ReactNode | ReactNode[];
```

Comment:
Why do we have a `ReactNode[]` here? Shouldn't be a single error?

Reply:
The EUI component `EuiComboBox` is the ultimate receiver of this props and it expects both option. Since this is a generic component, i made it expect the same.